scripts/v2/cipool/deploy.bicep

```diff
@@ -0,0 +1,78 @@
+param location string = resourceGroup().location
+
+var poolName = 'aso-1es-pool'
+var sku = 'Standard_D8ds_v4'
+
+// Base 1ES Image Resource IDs
+var ubuntu2204GalleryVersionResourceId = '/subscriptions/723b64f0-884d-4994-b6de-8960d049cb7e/resourceGroups/CloudTestImages/providers/Microsoft.Compute/galleries/CloudTestGallery/images/MMSUbuntu22.04-Secure/versions/latest'
+
+
+var poolSettings = {
+  maxPoolSize: 4 // We run 2 jobs per CI run on this pool, so a max of 4 means we can run 2 CI builds in parallel
+  resourcePredictions: [
+    {
+      '21:00': 2  // 9 AM Monday NZT
+    }
+    {
+      '05:00': 0  // 5 PM Monday NZT
+      '16:00': 2  // 9 AM Monday PST
+    }
+    {
+      '05:00': 0  // 5 PM Tuesday NZT
+      '16:00': 2  // 9 AM Tuesday PST
+    }
+    {
+      '05:00': 0  // 5 PM Wednesday NZT
+      '16:00': 2  // 9 AM Wednesday PST
+    }
+    {
+      '05:00': 0  // 5 PM Thursday NZT
+      '16:00': 2  // 9 AM Thursday PST
+    }
+    {
+      '05:00': 0  // 5 PM Friday NZT
+      '16:00': 2  // 9 AM Friday PST
```

Comment:
If the pool is at zero, does that mean no builds run? Or are they just slower because things need to be provisioned?

Reply:
This is AFAIK purely prewarming. So builds will take an additional 5-8m outside of these timeframes.